src/styles/styles.js

```diff
@@ -1417,11 +1417,17 @@ const styles = {
     },
 
     textInputComposeSpacing: {
-        paddingVertical: 6,
+        paddingVertical: 1,
+        alignSelf: 'center',
```

Comment:
Again adding `paddingVertical` with `alignSelf: 'center'`?

Are we sure it won't break anything on other platforms?

Reply:
I think we could move code to StyleUtils and define platform-specific styles.

cc: @Julesssss What do you think?

Reply:
> Are we sure it won't break anything on other platforms?

Yes, I did test out on other platforms!

Reply:
> I think we could move code to StyleUtils and define platform-specific styles.

Depends on whether the styling works for all platforms or not... Would prefer to have a single consistent style, if possible